components/admin_console/admin_definition.jsx

```diff
@@ -4808,17 +4807,10 @@ const AdminDefinition = {
                         defaultMessage='deprecated'
                     />
                 ),
-                shouldDisplay: (license) => license.IsLicensed && license.OpenId === 'true',
+                shouldDisplay: () => true,
             },
             isHidden: it.any(
-                it.any(
-                    it.not(it.licensed),
-                    it.licensedForSku('starter'),
-                ),
-                it.all(
-                    it.licensedForFeature('OpenId'),
-                    it.not(usesLegacyOauth),
-                ),
+                it.not(usesLegacyOauth),
```

Comment:
Just to confirm, we are changing oauth with this upgrade as well?

Reply:
As of now, yes: [reference](https://community.mattermost.com/private-core/pl/1n39ua1rytf9ddufwbncckrd1c).

Let's please talk more on the thread if we need to keep this separate.